jest.config.js

```diff
@@ -1,4 +1,4 @@
-const transformIgnorePatterns = ['node_modules/(?!(@openshift|lodash-es)/)'];
+const transformIgnorePatterns = ['node_modules/(?!(uuid)/)'];
```

Comment:
We really should remove this from the SDK...

Reply:
Yeah. Its a bit of a pain to deal with.